iOS/Flows/Conversation/Conversation List/ConversationListViewController+Extensions.swift

```diff
@@ -85,7 +85,9 @@ extension ConversationListViewController {
             }
         }.store(in: &self.cancellables)
         
-        ConversationsManager.shared.$activeConversation.mainSink { conversation in
+        ConversationsManager.shared.$activeConversation.mainSink { [weak self] conversation in
+            guard let `self` = self else { return }
+
```

Comment:
Conversation manager is a singleton so self was never getting released. This caused major memory leaks whenever we deep linked.